ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -12,6 +12,16 @@
 
 {%     if aaa_authorization.config_commands is defined and aaa_authorization.config_commands == true %}
 Authorization for configuration commands is enabled.
+| Type | User Stores |
+| ---- | ----------- |
+{%         if aaa_authorization.commands.all_default is defined %}
```

Comment:
Use arista.avd.defined everytime to make sure that the variable is defined and is not null/empty